kotsadm/operator/pkg/client/deploy.go

```diff
@@ -44,18 +47,23 @@ func (c *Client) diffAndRemovePreviousManifests(applicationManifests Application
 		return errors.Wrap(err, "failed to decode manifests")
 	}
 
+	targetNamespace := c.TargetNamespace
+	if applicationManifests.Namespace != "." {
```

Comment:
!= `"."`, or != `""`?

Reply:
I see that that's what it was before, but it feels odd

Reply:
ya, idk. but its correct